book/to-org/metrics-protocol.md

```diff
@@ -6,7 +6,7 @@ Our Metrics Protocol was designed to work with an external application to handle
 
 ## How are Metrics collected in Wallaroo?
 
-In order to have an understanding of the structure of a Metrics Message, you should know how we collect metrics in Wallaroo. Metrics in Wallaroo are collected for 3 different categories: computations, pipelines, and workers. A reporter is created for each source, sink, and step within a Wallaroo application. Depending on where and how a message is processed, it will use an associated reporter in order to record metrics for that message. Metrics are recorded as work is processed within a given time window. As soon as a new metric is recorded outside of our time window, a Metrics Message will be created and sent off from that reporter and we'll start recording for the next time window. Why does this matter? We will receive multiple Metrics Message's for a given item within a category, depending on how many reporters there are for that item. Time windows are used because there is the chance that we won't process a message for a few window cycles and this allows us to send off metrics belonging to an older time window regardless of when the latest metric arrives.
+In order to have an understanding of the structure of a Metrics Message, you should know how we collect metrics in Wallaroo. Metrics in Wallaroo are collected for 3 different categories: computations, pipelines, and workers. A reporter is created for each source, sink, and coalesced sequence of computations within a Wallaroo application. Depending on where and how a message is processed, it will use an associated reporter in order to record metrics for that message. Metrics are recorded as work is processed within a given time window. As soon as a new metric is recorded outside of our time window, a Metrics Message will be created and sent off from that reporter and we'll start recording for the next time window. Why does this matter? We will receive multiple Metrics Message's for a given item within a category, depending on how many reporters there are for that item. Time windows are used because there is the chance that we won't process a message for a few window cycles and this allows us to send off metrics belonging to an older time window regardless of when the latest metric arrives.
```

Comment:
"We will receive multiple Metrics Message's" -> "We will receive multiple Metrics Messages"